spyrmsd/graph.py

```diff
@@ -1,42 +1,97 @@
+from spyrmsd import constants
+
+import numpy as np
+import os
+
+available_backends = []
```

Comment:
```suggestion
_available_backends = []
```

I think this should be a private variable of the module, not part of the public API.